src/genworld.cpp

```diff
@@ -329,5 +329,8 @@ void GenerateWorld(GenWorldMode mode, uint size_x, uint size_y, bool reset_setti
 	/* Centre the view on the map */
 	ScrollMainWindowToTile(TileXY(Map::SizeX() / 2, Map::SizeY() / 2), true);
 
+	/* Safety check in case any initialization touched random. */
```

Comment:
This comment says more about what it does, which I can read on the next line too. It kinda doesn't tell me why it is doing that. Is there something we can add here to make it a bit more clear why it is important the initializers don't use random? (an actual question; as I have tried but failed to come up with something that works :P ).